.github/workflows/reusable_pre_release.yaml

```diff
@@ -44,6 +44,17 @@ on:
         default: msi
         required: false
         type: string
+      goarch_matrix:
+        description: "Defines a custom goarch-matrix for windows builds as nrjmx based integrations support amd64 only"
```

Comment:
Usually actions take the arguments as comma-separated list.

NIT: Leave a comment stating that it needs to be in JSON format

Reply:
If it iw for Windows only, I would also change the name of the argument from `goarch_matrix` to `windows_build_goarch_matrix`.

Reply:
I've added the comment and updated the names, thanks! 🙂